material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -2014,3 +2015,143 @@ p.quick-ref-title {
   height: unset;
   padding: 1em 2em;
 }
+
+/* Cookie Consent Styling */
+.md-consent__inner {
+  z-index: 999;
+}
+
+form.md-consent__form.md-grid.md-typeset {
+  border: var(--border-width-thin) solid var(--light-transparent-15);
+  display: flex;
+  align-items: center;
+  padding: 1.5rem;
+  border-radius: 8px;
+  max-width: 90%;
+  margin: 2em auto;
+}
+
+/* Logo Section */
+.cookie-logo img {
```

Comment:
I noticed in moonbeam.network they set the container for the logo to 210px; so let's add this CSS to follow suit and ensure that the logo is centered:

```
.cookie-logo {
  width: 210px;
  display: flex;
  justify-content: center;
  margin: auto;
}
```

Reply:
Actually, let's use responsive styles so we could make this like `13em`